recipes/approvaltests.cpp/8.0.0/test_package/test_doctest.cpp

```diff
@@ -0,0 +1,6 @@
+#define APPROVALS_DOCTEST
```

Comment:
According to the docs this is only needed for `Catch2`: https://github.com/approvals/ApprovalTests.cpp/blob/master/doc/GettingStarted.md#the-basics

```suggestion
```

Reply:
This code is correct, and should not be changed.

The example says that `#define APPROVALS_CATCH` (note the `CATCH`) is needed for Catch.

I see this file is called test_doctest.cpp, so it is correct that it is using `#define APPROVALS_DOCTEST` - which instructs Approvals to enable the doctest integration.